### PR TITLE
issue(#4): Add intro to home

### PR DESCRIPTION
**Pull request by naseerkassar**
please check the code and provide a feedback if there is anything needed to be fixed .
note: this issue was labelled as not ready i did the analysis if there is  an issue please provide feedback thanks :)

### CODE REVIEW

File: src/pages/index.jsx [hunk 1]
```diff
@@ -50,15 +56,19 @@ export const query = graphql`
...
 export default function IndexPage({
   data: {
+    homeIntro: { nodes: homeIntro },
     allApproaches: { nodes: allApproaches },
     firstCases: { nodes: firstCases },
     firstMembers: { nodes: firstMembers },
     firstPosts: { nodes: firstPosts },
   },
 }) {
+    console.log(homeIntro);
```
Comment (Moeriki) [inline src/pages/index.jsx]: 🙅‍♂️

File: src/pages/index.jsx [hunk 2]
```diff
@@ -10,6 +10,12 @@ import MembersGridList from '../members/grid-list';
 
 export const query = graphql`
   query IndexQuery($language: String!) {
+    homeIntro: allWordpressPage(filter: {title: {eq: "Home"}, language: { eq: $language } }
```
Comment (Moeriki) [inline src/pages/index.jsx]: Should be `wordpressPage`.

File: src/pages/index.jsx [hunk 3]
```diff
@@ -50,15 +56,19 @@ export const query = graphql`
...
     firstCases: { nodes: firstCases },
     firstMembers: { nodes: firstMembers },
     firstPosts: { nodes: firstPosts },
   },
 }) {
+    console.log(homeIntro);
   return (
     <Layout>
       <SEO title="Home" />
+      <h3>Foursevens</h3>
```
Comment (Moeriki) [inline src/pages/index.jsx]: No need for this to be an H3. Just `div` is probably fine.

File: src/pages/index.jsx (hunk 1 above)
Comment (naseerkassar) [reply src/pages/index.jsx]: fixed sorry :grimacing:

File: src/pages/index.jsx (hunk 2 above)
Comment (naseerkassar) [reply src/pages/index.jsx]: i needed all pages to filter by language in wordpressPage there is no language

File: src/pages/index.jsx (hunk 3 above)
Comment (naseerkassar) [reply src/pages/index.jsx]: i made it a div as you requested but i saw it in our website it was h3 so i did it the same

File: src/pages/index.jsx (hunk 2 above)
Comment (Moeriki) [reply src/pages/index.jsx]: This works.

```js
query MyQuery {
  wordpressPage(language: {eq: "nl"}) {
    title
  }
}
```

Comment (Moeriki) [reply src/pages/index.jsx]: Note that the filter arg works different on single-item queries! It's not wrapped in `filter: {}`

Comment (naseerkassar) [reply src/pages/index.jsx]: indeed fixed now thanks 👍

**Pull request merged**